dashboard/config/foorm/library/surveys/pd/workshop_elements.0.json

```diff
@@ -29,6 +29,125 @@
          "type": "html",
          "name": "thank_you_heading",
          "html": "<h2>Thank you!</h2>"
+       },
+       {
```

Comment:
I added this header element which we didn't end up actually using. Might want to switch it in though, so might as well leave it as an option.

Reply:
agree there's no harm keeping it in